packages/mds-audit-api/tests/api.spec.ts

```diff
@@ -583,7 +586,12 @@ describe('Testing API', () => {
 
       const seedData = {
         // Include a duplicate device (same vin + provider but different device_id)
-        devices: [...devices_a, ...devices_b, ...devices_c, { ...(devices_c[0] as Device), ...{ device_id: uuid() } }],
+        devices: [
+          ...devices_a,
+          ...devices_b,
+          ...devices_c,
+          { ...(devices_c[0] as Device), ...{ device_id: uuid() } }
```

Comment:
Should probably be able to change this one too right?

```suggestion
          { ...(devices_c[0] as DeviceDomainModel), ...{ device_id: uuid() } }
```